src/components/sections/Hero.astro

```diff
@@ -18,14 +18,14 @@ import plasmaScreenshotDark from "../../assets/42/screenshots/plasma-dark.webp";
       <h1
         class="hero-text font-bold text-4xl lg:text-5xl sm:text-6xl mb-2 leading-loose block"
       >
-        Do your thing
+        Do your thing.
       </h1>
     </div>
     <span class="inline-block lg:text-lg max-w-xl 2xl:max-w-2xl text-gray-200">
-      Ultramarine is a modern operating system that lets you just do your thing
-      with the hardware and tools you already love. For work, school, and play.
+      Ultramarine Linux is a modern operating system that supports your workflow
+      with the hardware and tools you already love, for work, school, and play.
       However you use your system, we have you covered with tweaks and tools to
-      make things run smoother.
+      make things run more seamlessly out of the box.
```

Comment:
These seem like synonyms, additionally, maybe we should reword "things", something like "make your work seemless" ?